otbox.py

```diff
@@ -97,6 +108,36 @@ def __init__(self):
         )
         self.mqttthread.start()
 
+
+    def set_params(self):
+        # sets the value of BAUDRATE, BROKER_ADDRESS, MOTE_USB_DEVICES, FIRMWARE_EUI64_RETRIEVAL, FIRMWARE_TEMP
+        # based on the selected testbed
+        global BAUDRATE
```

Comment:
Why BAUDRATE is a global value rather than a value of otbox class?

Reply:
`BAUDRATE` variable is used by two classes: `OtBox` and `SerialportHandler`. Since the instances of these classes can exist independently of one another it's best to keep `BAUDRATE` as a global variable.